src/element/textWysiwyg.tsx

```diff
@@ -434,61 +435,54 @@ export const textWysiwyg = ({
     // it'd get stuck in an infinite loop of blur→onSubmit after we re-focus the
     // wysiwyg on update
     cleanup();
-    const updateElement = Scene.getScene(element)?.getElement(element.id);
+    const updateElement = Scene.getScene(element)?.getElement(
+      element.id,
+    ) as ExcalidrawTextElement;
     if (!updateElement) {
       return;
     }
-    let wrappedText = "";
-    if (isTextElement(updateElement) && updateElement?.containerId) {
-      const container = getContainerElement(updateElement);
-
-      if (container) {
-        wrappedText = wrapText(
-          editable.value,
-          getFontString(updateElement),
-          container.width,
-        );
+    let text = editable.value;
```

Comment:
This is just for spec, for some reason in specs If `change` event is triggered, its failing when `updatingBoundElements` in one of the specs, tried debugging it but no luck yet so keeping it `editable.value` as before for unbounded text